pysrc/riffq/__init__.py

```
@@ -0,0 +1,6 @@
+from ._riffq import Server  # Rust class
```

Ruff (0.11.9)

`._riffq.Server` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)